hypha/apply/funds/workflow.py

```diff
@@ -343,11 +343,12 @@ def make_permissions(edit=None, review=None, view=None):
                 'ext_internal_review': _('Open Review'),
                 'ext_determination': _('Ready For Determination'),
                 'ext_rejected': _('Dismiss'),
+                'ext_screening_withdrawn': _('Withdraw'),
             },
             'display': _('Need screening'),
             'public': _('Application Received'),
             'stage': RequestExt,
-            'permissions': default_permissions,
+            'permissions': applicant_edit_permissions,
```

Comment:
I think withdraw permissions should be separate from edit permissions.

We do not allow applicants to edit submissions that are in review for obvious reasons.

But if we allow an applicant to withdraw an application they should be able to do that in any state up until the submission is approved/declined.

Reply:
In progress.

Reply:
I think https://github.com/HyphaApp/hypha/pull/3298/commits/f4ae00168a4651d18a6fe9420a54750f1bdf0480 resolves this, let me know what you think.